modules/data-entry/src/main/frontend/src/vocabQuery/InfoBox.jsx

```diff
@@ -0,0 +1,129 @@
+        },
+        arrow: {
+          enabled: false
+        }
+      }}
+    >
+      {({ TransitionProps }) => (
+        <Grow
+          {...TransitionProps}
+          id="info-grow"
+          style={{
+            transformOrigin: "center left",
+          }}
+        >
+          <Card className={classes.infoCard}>
+            <ClickAwayListener onClickAway={onClickAway}><div>
+               <CardHeader
+                 avatar={
+                  <Link color="textSecondary"
+                    href={vocabulary.url || ""}  target="_blank"
+                    component={vocabulary.url ? 'a' : 'span'}
+                    underline="none"
+                    >
+                    <Tooltip title={vocabulary.description || ""}>
+                      <Avatar aria-label="source" className={classes.vocabularyAvatar}>
+                          {vocabulary.acronym}
+                      </Avatar>
+                    </Tooltip>
+                  </Link>
+                }
+                action={
+                  <IconButton aria-label="close" onClick={onClose}>
+                    <CloseIcon />
+                  </IconButton>
+                }
+                title={term.name}
+                subheader={term.id}
+                titleTypographyProps={{variant: 'h5'}}
+              />
+              <CardContent className={classes.infoPaper}>
+                <div className={classes.infoSection}>
+                  <Typography className={classes.infoDefinition}>{term.definition}</Typography>
+                </div>
+                  {term.alsoKnownAs.length > 0 && (
+                    <div className={classes.infoSection}>
+                      <Typography variant="h6" className={classes.infoHeader}>Also known as</Typography>
+                      {term.alsoKnownAs.map((name, index) => {
+                        return (<Typography className={classes.infoAlsoKnownAs} key={index}>
+                                  {name}
+                                </Typography>
+                        );
+                      })}
+                    </div>
+                  )}
+                  {term.typeOf.length > 0 && (
+                    <div className={classes.infoSection}>
+                      <Typography variant="h6" className={classes.infoHeader}>Is a type of</Typography>
+                      {term.typeOf.map((name, index) => {
+                        return (<Typography className={classes.typeOf} key={index}>
+                                  {name}
+                                </Typography>
+                        );
+                      })}
+                    </div>
+                  )}
+                  </CardContent>
+                  <CardActions className={classes.infoPaper + (browserOpened ? " " + classes.hiddenButton : '')}>
+                      <Button size="small" onClick={openBrowser} variant='contained' color='primary'>Learn more</Button>
+                  </CardActions>
```

Comment:
I never liked the fact that the "Learn more" button disappeared in the background when the browser was open, and now we have an opportunity to improve it. We're not supposed to show "Learn more" when displaying the info for a term in the Vocabulary Browser, but it should not affect what the info box for a VocabularyQuery suggestion looks like.

Rather than hide the button when the browser opens, we should only display it if info box is not above background.

Would this work?
```suggestion
                  <CardActions className={classes.infoPaper}>
                    { !(broswerOpened && infoAboveBackground) &&
                      <Button size="small" onClick={openBrowser} variant='contained' color='primary'>Learn more</Button>
                    }
                  </CardActions>
```

Reply:
Addressed with changes (hid the whole DialogActions, fixed typo in var name) in 309a2e29.